### DISCUSSION TITLE
OrientDB new version not starting.

### DISCUSSION THREAD

**Discussion by shaantheadmin** [orientechnologies/orientdb#9609] [General] [answered]
I am trying to setup OrientDB for a new project. surprisingly latest version of orientdb-3.2.0 is not starting the service and struck at OSystemUserAuthenticator. I tried to run on ubuntu 18 and also 20.04. I am using Java 11. Can some one please guide...

**Comment by johny65** [comment, marked as answer]
If you are still facing this, here is a solution: https://github.com/orientechnologies/orientdb/issues/9618.
